components/clp-package-utils/clp_package_utils/scripts/search.py

```
@@ -87,6 +97,13 @@ def main(argv):
         )
         return -1
 
+    dataset = parsed_args.dataset
+    if StorageEngine.CLP_S == storage_engine:
+        dataset = CLP_DEFAULT_DATASET_NAME if dataset is None else dataset
+    elif dataset is not None:
+        logger.error(f"Dataset selection is not supported for storage engine: {storage_engine}.")
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)